css/style.css

```diff
@@ -69,15 +71,153 @@ body {
 header {
 
 }
+h1 {
+  font-size: 24px;
+  font-weight: 900;
+  color: white;
+  padding-top:50px;
+  padding-left: 300px;
+}
+h2 {
+  color: white;
+  font-size: 24px;
+  font-weight: 500;
+  /*float: center;*/
+  padding-left:32%;
+  margin:30px;
 
-footer {
+}
+.GetItNow {
+  background-color: #f9e42e;
+  font-weight: 900;
+  font-size: 14px;
+  border-radius: 5px;
+  height:35px;
+  width:150px;
+  margin-left:600px;
+  border: none;
+  color: #033048;
+}
 
+footer {
+  text-align:center;
+ }
+.footer-text {
+  font-size: 9px;
+  font-weight: 500;
+  color:white;
+  margin:10px;
 }
 
 /******************************************
 /* ADDITIONAL STYLES
 /*******************************************/
 
+.second {
+  margin-top:50px;
+  margin-bottom: 50px;
+  background-color: #eee;
+  height:330px;
+  padding-top: 15px;
+
+}
+h3 {
+  font-size: 20px;
+  font-weight: 600;
+  margin-left: 100px;
+  color:#033048;
+  margin-bottom: 20px;
+  /*margin-top:50px;*/
+}
 
+.grey {
+  color:#606161;
+  margin-left: 100px;
+  font-size:13px;
+  font-weight:200;
+  /*margin-bottom: 20px;*/
+}
+p ~ ul {
+  margin-left: 115px;
+}
+p ~ p {
+  margin-bottom: 20px;
+}
 
+.quote {
+  font-style: italic;
+  font-weight: 300;
+  font-size: 18px;
+  color: white;
+  
 
+}
+.quoter {
+  font-size: 18px;
+  font-weight: 600;
+  color: white;
+}
+ .last { 
+  margin-top: 100px;
+  margin-bottom: 100px;
+  margin-left: 300px;
+  margin-right: 300px;
+ }
+
+ .yellow {
+  height:100px;
+  background-color: #f9e42e;
+  padding:30px;
+  /*padding-right:5000px ;*/
+  /*padding-right: 30%;*/
+  /*padding-left: 30%;*/
+ }
+ .black {
+  height:400px;
+  background-color: black;
+ }
+
+ .first-button {
+  margin-bottom: 400px;
+
+ }
+ /*.second-button-wrap{
+    background-color: #033048;
+    font-weight: 900;
+    font-size: 12px;
+    border-radius: 5px;
+    color:white;
+    text-align: center;
+    height:35px;
+    width:150px;
+    margin-left: 600px;
+    /*margin-right: 10000px;*/
+ 
+ .second-button{
+  background-color: #033048;
+  font-weight: 900;
+  font-size: 12px;
+  border-radius: 5px;
+  height:35px;
+  width:150px;
+  margin-left:600px;
+  border: none;
+  color: white;
+}
+ .relaxr {
+  color:#f9e42e;
+  font-weight: 900;
+  font-size:24px;
+  text-align: center;
+  padding-top: 60px;
+  padding-bottom:60px;
+ }
+ .social{
+  padding-left: 625px;
```

Comment:
:hammer: I see that you're trying to center the social icons. This might look good on you're screen (depending on the resolution), but if you shrink the browser a bit or someone else is looking with a bigger screen, it won't be positioned correctly. Instead of padding, try putting a width on this element and adjusting the margin "automatically" to get it centered on the page just right. Do you remember the margin centering trick? :)